scripts/rabbitmq-server

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh -e
+#!/bin/bash -e
```

Comment:
You can't hardcode the path to Bash because it could be installed somewhere else. Please use `/usr/bin/env bash` instead.

In the process, you'll have to remove the `-e` and add a `set -e` (which is also cleaner) after the license.